### PR TITLE
Added broker example

### PR DESCRIPTION
**Pull request by jroper**

### CODE REVIEW

**Comment by jroper** [comment]
I added travis validation to this PR - this required moving all the files to `src/main/g8` (which g8 will automatically use if it exists) so that I could add a `travis.yml` that wouldn't be included in the template itself. The validation both checks the default configuration, and also using a custom project name and organisation, to ensure everything is correctly templated.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    - SCRIPT="sbt new file://$(pwd) --name=smello --organization=org.eggsample && cd smello && sbt test"
+script: eval $SCRIPT
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/scala*
+before_cache:
+  # Ensure changes to the cache aren't persisted
+  - rm -rf $HOME/.ivy2/cache/com.lightbend.lagom/*
+  - rm -rf $HOME/.ivy2/cache/scala_*/sbt_*/com.lightbend.lagom/*
```
Comment (TimMoore) [inline .travis.yml]: Do we really want to remove Lagom from the Ivy cache in this project?

Comment (jroper) [inline .travis.yml]: You're right.

**Comment by TimMoore** [comment]
@jroper do you plan to make the same change in the Scala template?

**Comment by jroper** [comment]
Yep, but let's make sure that GSA is happy with it first.

**Comment by ignasi35** [comment]
We need to promote changes on the maven archetype too.

**Pull request merged**